scipy-tutorial/06_xarray_and_dask.ipynb

```diff
@@ -17,9 +17,58 @@
   },
```

Comment:
<p>Reminder to clear all output. We added <code>pre-commit</code>  over the weekend so <code>pre-commit run --all-filespre-commit run --all-files</code>  will do it for you.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/xarray-contrib/xarray-tutorial/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>